Mathlib/LinearAlgebra/TensorProduct.lean

```diff
@@ -801,6 +802,26 @@ lemma map_comp_comm_eq (f : M →ₗ[R] P) (g : N →ₗ[R] Q) :
       TensorProduct.comm R Q P ∘ₗ map g f :=
   ext rfl
 
```

Comment:
I think it's worth adding comments to this lemma along the lines of:
```suggestion

/--  Given linear maps `f : M → Q`, `g : N → S`, and `h : P → T`, if we identify `(M ⊗ N) ⊗ P` with
`M ⊗ (N ⊗ P)`, and `(Q ⊗ S) ⊗ T` with `Q ⊗ (S ⊗ T)` then this lemma states that
`f ⊗ (g ⊗ h) = (f ⊗ g) ⊗ h`. -/
```
and likewise its three friends below.